fuzzers/063-gtp-common-conf/Makefile

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2017-2020  The Project X-Ray Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+SHELL = bash
+
+N ?= 20
+
+BUILD_DIR = build_${XRAY_PART}
+
+SPECIMENS := $(addprefix ${BUILD_DIR}/specimen_,$(shell seq -f '%03.0f' $(N)))
+SPECIMENS_OK := $(addsuffix /OK,$(SPECIMENS))
+FUZDIR ?= ${PWD}
+
+
+all: database
+
+# generate.sh / top_generate.sh call make, hence the command must
+# have a + before it.
```

Comment:
This comment indicates that `top_generate.sh` line should have the `+`, but it is not present?